packages/EasyTest/src/Stub/HttpClient/HttpClientAssertionTrait.php

```diff
@@ -4,28 +4,12 @@
 
 namespace EonX\EasyTest\Stub\HttpClient;
 
-use PHPUnit\Framework\Assert;
-use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
-use Symfony\Contracts\HttpClient\HttpClientInterface;
+use EonX\EasyTest\Traits\HttpClientAssertionTrait as BaseTrait;
 
+/**
+ * @deprecated since 4.3.5, will be removed in 4.3.6. Use EonX\EasyTest\Traits\HttpClientAssertionTrait instead.
```

Comment:
Please say that we remove this in version `5.0.0`.

Reply:
Done

Reply:
thanks